src/Controls/tests/TestCases.HostApp/Issues/Issue13558.xaml.cs

```diff
@@ -0,0 +1,117 @@
+﻿using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Xaml;
+using Microsoft.Maui.Platform;
+using Microsoft.Maui.Controls.PlatformConfiguration;
+using Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific;
+using System;
+using System.ComponentModel;
+using System.Collections.ObjectModel;
+using System.Runtime.CompilerServices;
+
+namespace Maui.Controls.Sample.Issues;
+
+[XamlCompilation(XamlCompilationOptions.Compile)]
+[Issue(IssueTracker.Github, 13558, "Picker values are not displaying when loaded within a ListView", PlatformAffected.Android)]
+
+public partial class Issue13558 : ContentPage
+{
+	public ObservableCollection<List<string>> Items { get; set; }
+
+	MainViewModel _mainViewModel;
+	public Issue13558()
+	{
+		InitializeComponent();
+		_mainViewModel = new MainViewModel();
+		BindingContext = _mainViewModel;
+	}
+}
+
+public class BloodworkStatus : INotifyPropertyChanged
+{
+	public int Id { get; set; }
+	public string Name { get; set; }
+
+	public event PropertyChangedEventHandler PropertyChanged;
+
+	public override string ToString()
+	{
+		return Name!.ToString();
+	}
+
+	private void NotifyPropertyChanged([CallerMemberName] String propertyName = "")
+	{
+		PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+	}
+}
+
+public class MainViewModel : INotifyPropertyChanged
```

Comment:
The build is failing:
```
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25191.xaml.cs(15,15): error CS0101: The namespace 'Maui.Controls.Sample.Issues' already contains a definition for 'MainViewModel' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-maccatalyst]
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25191.xaml.cs(15,15): error CS0101: The namespace 'Maui.Controls.Sample.Issues' already contains a definition for 'MainViewModel' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-android]
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25191.xaml.cs(15,15): error CS0101: The namespace 'Maui.Controls.Sample.Issues' already contains a definition for 'MainViewModel' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-ios]
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25191.xaml.cs(15,15): error CS0101: The namespace 'Maui.Controls.Sample.Issues' already contains a definition for 'MainViewModel' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-tizen]
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25191.xaml.cs(15,15): error CS0101: The namespace 'Maui.Controls.Sample.Issues' already contains a definition for 'MainViewModel' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.19041.0]
D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25191.xaml.cs(15,15): error CS0101: The namespace 'Maui.Controls.Sample.Issues' already contains a definition for 'MainViewModel' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.20348.0]
    6924 Warning(s)
    6 Error(s)
```

There is another sample using a MainViewModel class. Could you rename the view model to include the issue id?
Something like `Issue13558ViewModel`

Reply:
> The build is failing:
> 
> ```
> D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25191.xaml.cs(15,15): error CS0101: The namespace 'Maui.Controls.Sample.Issues' already contains a definition for 'MainViewModel' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-maccatalyst]
> D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25191.xaml.cs(15,15): error CS0101: The namespace 'Maui.Controls.Sample.Issues' already contains a definition for 'MainViewModel' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-android]
> D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25191.xaml.cs(15,15): error CS0101: The namespace 'Maui.Controls.Sample.Issues' already contains a definition for 'MainViewModel' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-ios]
> D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25191.xaml.cs(15,15): error CS0101: The namespace 'Maui.Controls.Sample.Issues' already contains a definition for 'MainViewModel' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-tizen]
> D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25191.xaml.cs(15,15): error CS0101: The namespace 'Maui.Controls.Sample.Issues' already contains a definition for 'MainViewModel' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.19041.0]
> D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Issues\Issue25191.xaml.cs(15,15): error CS0101: The namespace 'Maui.Controls.Sample.Issues' already contains a definition for 'MainViewModel' [D:\a\_work\1\s\src\Controls\tests\TestCases.HostApp\Controls.TestCases.HostApp.csproj::TargetFramework=net9.0-windows10.0.20348.0]
>     6924 Warning(s)
>     6 Error(s)
> ```
> 
> There is another sample using a MainViewModel class. Could you rename the view model to include the issue id? Something like `Issue13558ViewModel`

Hi @jsuarezruiz  Following your suggestion, the class name has been updated to
Issue13558ViewModel. Please let us know if you have any further concerns.